### DISCUSSION TITLE
Possible to hide the add button on an join type field?

### DISCUSSION THREAD

**Discussion by ckruppe** [payloadcms/payload#8677] [Q&A] [answered]
Hi at all,

as the Title states, is it possible to remove the "add new" button from an join field in the admin ui?
![Screenshot 2024-10-13 174452](https://github.com/user-attachments/assets/84cb71a5-3eca-47ee-8438-24381cf321fb)

**Comment by akhrarovsaid** [comment, marked as answer]
Hey @ckruppe,

It seems at this time (beta.113) that there's no way to disable the `Add New` button - with one exception: that the user _does not have_ the required permissions to create a new entry in the collection that this join field has a relation to. See [the line that impacts this here](https://github.com/payloadcms/payload/blob/beta/packages/ui/src/elements/RelationshipTable/index.tsx#L205).

**Reply by ckruppe** [reply]
Ah i see. Thanks, then i will leave it like it is right now. :)
